src/components/Nav.svelte

```diff
@@ -23,19 +24,22 @@
     {/if}
 
     {#if open || !mobile}
```

Comment:
Would the `|| !mobile` still be needed? You changed line 10 to add !mobile

Reply:
I believe it's still needed unless you **a**: initialize open as true and (maybe*) **b**: check if mobile `onMount` and set accordingly because line 10 only runs when mobile changes (*unless it will trigger itself automatically on init, I'm not 100% sure).

Reply:
Will do some testing